src/main/java/eu/interop/federationgateway/service/DiagnosisKeyCleanupService.java

```diff
@@ -59,8 +62,17 @@ public void cleanupDiagnosisKeys() {
 
     log.info("Starting DiagnosisKey and DiagnosisKeyBatch cleanup");
 
+    final int keysBefore =  (int) diagnosisKeyEntityService.countAll();
     final int deletedDiagnosisKeys = diagnosisKeyEntityService.deleteAllBefore(deleteTimestamp);
     final int deletedDiagnosisKeyBatches = diagnosisKeyBatchService.deleteAllBefore(deleteTimestamp);
```

Comment:
We should not remove this deletion of the batches, because now we need them for keeping statistics. This process should only delete the keys.

Reply:
Done